DeepSpeech.py

```diff
@@ -141,11 +141,12 @@ def rnn_impl_static_rnn(x, seq_length, previous_state, reuse):
     return output, output_state
 
 
-def create_model(batch_x, seq_length, dropout, reuse=False, previous_state=None, overlap=True, rnn_impl=rnn_impl_lstmblockfusedcell):
+def create_model(batch_x, batch_size, seq_length, dropout, reuse=False, previous_state=None, overlap=True, rnn_impl=rnn_impl_lstmblockfusedcell):
     layers = {}
 
     # Input shape: [batch_size, n_steps, n_input + 2*n_input*n_context]
-    batch_size = tf.shape(batch_x)[0]
+    if not batch_size:
```

Comment:
Looks fine, I'm just wondering why we need that now ?

Reply:
We support exporting a model with dynamic batch size, in which case `batch_size` in this function has to be computed dynamically too. That would re-introduce the StridedSlice dependency, but since we don't support the dynamic batch size case in our clients, it's fine. Just trying not to break other people's use cases even if we don't use them ourselves.